docs/pages/router/reference/troubleshooting.mdx

```diff
@@ -5,6 +5,16 @@ description: Fixing common issues with Expo Router setup.
 
 import { Terminal } from '~/ui/components/Snippet';
 
+## Missing files / source maps in React Native DevTools
+
+This can happen if your Chrome DevTools has exclusions within its ignore list. To fix the issue
+
+1. Start the React Native DevTools
+2. Open settings by pressing the gear icon
+3. Press "restore defaults and reload"
```

Comment:
```suggestion
3. Under **Extensions**, click **Restore defaults and reload**
```

We use complete specific settings path and use bold text to highlight the settings and its menu item name in our docs. Let's stick to that.